docs/source/development.rst

```diff
@@ -30,3 +30,53 @@ the bundled and minified Javacript by using the option
 number of requests that the browser make to the server, but can allow to test
 Javascript file modification without going through the compilation step that
 can take up to 30 sec.
+
+
+Making a notebook release
+-------------------------
+
+Make sure you have followed the step above and have all the tools to genereate
+the minified javascript and css files. 
+
+Make sure the repository is clean of any file that could be problematic. 
+You can remove all non-tracked files with:
+
+.. code::
+
+    $ git clean -xfdi
+
+This would ask you for confirmation before removing all untracked file. Make
+sure the ``dist/`` folder in particular is clean and avoid stale build from
```

Comment:
... folder ~~in particular~~ is clean and remove any stale files from previous build attempts.